src/tasks/leveling.ts

```diff
@@ -124,6 +124,13 @@ const craftedCBBFoods: Item[] = $items`honey bun of Boris, roasted vegetable of
 const craftedCBBEffects: Effect[] = craftedCBBFoods.map((it) => effectModifier(it, "effect"));
 let triedCraftingCBBFoods = false;
 
+const LOVEquip =
+  mainStatStr === $stat`Muscle`
+    ? "LOV Eardigan"
+    : mainStatStr === $stat`Mysticality`
+      ? "LOV Epaulettes"
+      : "LOV Earring";
```

Comment:
This is good, but also probably warrants the change you mentioned in `outfit.ts` - especially since the epaulettes are "hardcoded" in there as well.